File: ios/Doximity.m
//
//  Doximity.m
//  Doc Hello
//
//  Created by Suhel on 14/04/2023.
//

#import <Foundation/Foundation.h>

#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(DocHelloDoximityDialer, NSObject);
RCT_EXTERN_METHOD(openDoximityDialer:(NSString *)phone)
@end

File: ios/Doximity.swift
//
//  Doximity.swift
//  Doc Hello
//
//  Created by Suhel on 14/04/2023.
//

import Foundation

@objc(DocHelloDoximityDialer)
class DocHelloDoximityDialer: NSObject {
  @objc
  func openDoximityDialer(_ phoneNumber: String) -> Void  {
    DoxDialerCaller.shared().dialPhoneNumber(phoneNumber);
  }
  @objc
  static func requiresMainQueueSetup() -> Bool {
      return true
  }
}
